lib/components/viewers/viewers.css

```diff
@@ -102,48 +102,86 @@
 
 /* stop viewer styles */
 
-.otp .stop-viewer .route-row {
```

Comment:
Any way we can set some form of min-height on this? To prevent this issue: 
![image](https://user-images.githubusercontent.com/86619099/223164147-9e9ca983-461b-4a7e-87a0-55914c6aa677.png)